### ISSUE TITLE
Question/Bug: using both callback_data and login_url into an InlineKeyboardButton

### ISSUE THREAD

**Issue by nichis** [yagop/node-telegram-bot-api#785]
I have added two InlineKeyboardButton with _login_url_ in a message. When I click on one of them the bot doesn't receive any inline_query message, so I added also the field _callback_data_ but in this way the field _login_url_ is not working anymore.

Are these two fields not compatible togheter?

**Comment by cankilinc** [comment]
you can do this with callback requery like this

bot.on('callback_query', query => {}

**Comment by nichis** [comment]
> you can do this with callback requery like this
> 
> bot.on('callback_query', query => {}

yes of course, using the field _callback_data_ , but if I add also the field _login_url_ the event callback_query isn't triggered

**Comment by yagop** [comment]
Telegram requires exactly one action field on each `InlineKeyboardButton`. `callback_data` and `login_url` are therefore not compatible on the same button. See https://core.telegram.org/bots/api#inlinekeyboardbutton. Closing as answered.
